gammapy/cube/tests/test_simulate.py

```diff
@@ -43,7 +43,7 @@ def test_simulate():
     )
 
     assert isinstance(dataset, MapDataset)
-    assert isinstance(dataset.model, SkyModel)
+    assert isinstance(dataset.model, SkyModels)
```

Comment:
I'm a bit confused about the naming here. So `dataset.model` is a `SkyModels` object containing multiple models? Maybe `dataset.models` is clearer?

Reply:
I agree this might be clearer. I'll prepare a PR this week to clean up the `Dataset` classes before the v0.11 release, which will include a few other changes we've discussed (e.g. renaming `Dataset.likelihood()` to `Dataset.fit_statistic()` and improving the docstrings).